modules/rosa-hcp-sts-creating-a-cluster-cli-no-cni-plugin.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-cluster-no-cni.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-cli_{context}-no-cni"]
+= Creating the cluster
+
+When using the {product-title} (ROSA) command line interface (CLI), `rosa`, to create a cluster, you can add an optional flag `--no-cni` to create a cluster without a CNI plugin.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for {hcp-title}.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host. Run `rosa version` to see your currently installed version of the ROSA CLI. If a newer version is available, the CLI provides a link to download this upgrade.
+* You have logged in to your Red Hat account by using the ROSA CLI.
+* You have created an OIDC configuration.
+* You have verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
+
+.Procedure
+
+. You can create your {hcp-title} cluster with one of the following commands.
++
+[NOTE]
+====
+When creating a {hcp-title} cluster, the default machine Classless Inter-Domain Routing (CIDR) is `10.0.0.0/16`. If this does not correspond to the CIDR range for your VPC subnets, add `--machine-cidr <address_block>` to the following commands. To learn more about the default CIDR ranges for {product-title}, see xref:../networking/cidr-range-definitions.adoc#cidr-range-definitions[CIDR range definitions].
+====
++
+** Create a cluster with a single, initial machine pool, publicly available API, publicly available Ingress, and no CNI plugin by running the following command:
++
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=<cluster_name> \
+    --sts --mode=auto --hosted-cp --operator-roles-prefix <operator-role-prefix> \
+    --oidc-config-id <ID-of-OIDC-configuration> --subnet-ids=<public-subnet-id>,<private-subnet-id> --no-cni
+----
+
+** Create a cluster with a single, initial machine pool, privately available API, privately available Ingress, and no CNI plugin by running the following command:
++
+[source,terminal]
+----
+$ rosa create cluster --private --cluster-name=<cluster_name> \
+    --sts --mode=auto --hosted-cp --subnet-ids=<private-subnet-id> --no-cni
+----
+
+** If you used the `OIDC_ID`, `SUBNET_IDS`, and `OPERATOR_ROLES_PREFIX` variables to prepare your environment, you can continue to use those variables when creating your cluster without a CNI plugin. For example, run the following command:
```

Comment:
@jneczypor I think the purpose of this part should introduce how to use the environmental variables to create cluster.
The description is not very clear. We can refer to this part in other [chapter](https://github.com/openshift/openshift-docs/edit/main/modules/rosa-hcp-sts-creating-a-cluster-cli.adoc)

> If you set the environmental variables, create a cluster with a single, initial machine pool, a publicly available API, and a publicly available Ingress by running the following command:

Reply:
That's a great point! Our docs team plans to do a push throughout all of the documentation to use environmental variables. Let me add a comment and we can address this in another ticket in an upcoming sprint.